_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -0,0 +1,258 @@
+		accessible. Following these guidelines will make content more accessible to a wider range of people with
+		disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement,
+		speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities
+		and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines
+		address accessibility of web content on desktops, laptops, tablets, and mobile devices. Following these guidelines
+		will also often make Web content more usable to users in general.
+	</p>
+</section>
+```
+
+#### Failed Example 2
+
+This [horizontally scrollable][scrollable] `section` element is not included in [sequential focus navigation][], nor does it have any [descendants][descendant] that are.
+
+```html
+<style>
+	section {
+		height: 100px;
+		width: 400px;
+		overflow-y: auto;
+		white-space: nowrap;
+	}
+	section > img {
+		display: inline-block;
+		width: 80px;
+	}
+</style>
+<h1>Our sponsors:</h1>
+<section>
+	<img src="/test-asset/shared/w3c-logo.png" alt="W3C" />
+	<img src="/test-asset/shared/eu-logo.svg" alt="EU" />
+	<img src="/test-asset/shared/w3c-logo.png" alt="W3C" />
+	<img src="/test-asset/shared/eu-logo.svg" alt="EU" />
+	<img src="/test-asset/shared/w3c-logo.png" alt="W3C" />
+	<img src="/test-asset/shared/eu-logo.svg" alt="EU" />
+	<img src="/test-asset/shared/w3c-logo.png" alt="W3C" />
+</section>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This `section` element has a [computed][] [overflow][] of `visible`.
+
+```html
+<section style="height: 95px; width: 500px;">
+	<h1>WCAG 2.1 Abstract</h1>
+	<p>
+		Web Content Accessibility Guidelines (WCAG) 2.1 covers a wide range of recommendations for making Web content more
+		accessible. Following these guidelines will make content more accessible to a wider range of people with
+		disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement,
+		speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities
+		and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines
+		address accessibility of web content on desktops, laptops, tablets, and mobile devices. Following these guidelines
+		will also often make Web content more usable to users in general.
+	</p>
+</section>
+```
+
+#### Inapplicable Example 2
+
+This `section` element has a [scroll distance][scrollable] of 0 in both directions.
+
+```html
+<section style="height: 95px; width: 500px; overflow: auto;">
+	<p>
+		<a href="//www.w3.org/TR/WCAG21/#abstract">
+			WCAG 2.1 Abstract
+		</a>
+	</p>
+</section>
+```
+
+#### Inapplicable Example 3
+
+This `section` element has a [computed][] [overflow][] of `hidden`.
```

Comment:
```suggestion
This `section` element is not [scrollable][] because it has a [computed][] [overflow][] of `hidden`.
```
Applicability is about being scrollable, and only indirectly about overflow.